underfs/oss/pom.xml

```diff
@@ -40,8 +42,8 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
cleaning the deps. this `org.apache.httpcomponents` is declared but not used in oss module itself, according to `mvn dependency:analyze`